cogs/audio.py

```diff
@@ -290,6 +290,9 @@ def _cache_max(self):
         setting_max = self.settings["MAX_CACHE"]
         return max([setting_max, self._cache_min()])  # enforcing hard limit
 
+    def _noppl_disconnect(self):
+        setting_nodis = self.settings["NOPPL_DIS"]
```

Comment:
Yeah it will, Kowlin pulled some magic out of his ass a while back on load to make this happen